ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -170,24 +172,24 @@ module('Acceptance | secrets/secret/create, read, delete', function (hooks) {
       // navigate to farthest leaf
       await visit(`/vault/secrets/${enginePath}/list`);
       assert.dom('[data-test-component="navigate-input"]').hasNoValue();
-      assert.dom('[data-test-secret-link]').exists({ count: 1 });
-      await click('[data-test-secret-link="1/"]');
+      assert.dom(SS.secretLink()).exists({ count: 1 });
+      await click(SS.secretLink('1/'));
       assert.dom('[data-test-component="navigate-input"]').hasValue('1/');
-      assert.dom('[data-test-secret-link]').exists({ count: 2 });
-      await click('[data-test-secret-link="1/2/"]');
+      assert.dom(SS.secretLink()).exists({ count: 2 });
+      await click(SS.secretLink('1/2/'));
       assert.dom('[data-test-component="navigate-input"]').hasValue('1/2/');
-      assert.dom('[data-test-secret-link]').exists({ count: 1 });
-      await click('[data-test-secret-link="1/2/3/"]');
+      assert.dom(SS.secretLink()).exists({ count: 1 });
+      await click(SS.secretLink('1/2/3/'));
       assert.dom('[data-test-component="navigate-input"]').hasValue('1/2/3/');
-      assert.dom('[data-test-secret-link]').exists({ count: 2 });
+      assert.dom(SS.secretLink()).exists({ count: 2 });
```

Comment:
This PR is already big, so I wouldn't address this here - but it would be nice if we could replace all instances of the `SECRET_ENGINE_SELECTORS` (`SS`) with general selectors in a follow up pr.